manage.py

```diff
@@ -92,7 +92,7 @@ def setup_general():
                 email=Config.ADMIN_EMAIL)
             db.session.add(user)
             db.session.commit()
-            print 'Added administrator {}'.format(user.full_name())
+            print(('Added administrator {}'.format(user.full_name())))
```

Comment:
looks like there were two sets of parens added here and twice more further down in the file, I think only one set needs to be added?